sdk/cosmos/azure-cosmos/azure/cosmos/_query_iterable.py

```diff
@@ -52,45 +63,23 @@ def __init__(self, client, query, options, fetch_function, collection_link=None)
         >>>     return result['Databases']
 
         """
+        if continuation_token:
+            raise ValueError("Continuation token not supported by Query iterator.")
```

Comment:
Why can't we support a continuation token for query iterator? Is this because of fan-out of queries?

Reply:
Yes that's my understanding - I was advised by @christopheranderson that this should not be supported.

Reply:
Yeah, I'm not sure this is correct. I might have misinterpreted what was shown in the meeting. When we create the query, we can support a continuation token. x-partition queries won't return a continuation token for the time being until we port that (relatively massive) feature over to Python. So I do think we can support continuation tokens in this manner.